src/components/StakingVaults/AllEarnOpportunities.tsx

```diff
@@ -53,7 +90,7 @@ export const AllEarnOpportunities = () => {
         state: { background: location },
       })
     },
-    [dispatch, history, isConnected, location],
+    [dispatch, history, isConnected, location, cosmosStaking, cosmosGetStarted],
```

Comment:
Not sure about having these two as deps, we shouldn't be watching on them, modals from the `<ModalProvider />` should be referentially stable and we're not reacting on any change from those.
I'd rather silence the eslint than adding them as dependencies just to satisfy it.

Reply:
Removed deps and added an eslint ignore.